cirq-superstaq/cirq_superstaq/ops/qubit_gates_test.py

```diff
@@ -1102,6 +1102,61 @@ def test_itoffoli() -> None:
     )
 
 
+def test_dd_protocols() -> None:
+    with mock.patch("cirq.testing.consistent_protocols.assert_qasm_is_consistent_with_unitary"):
+        cirq.testing.assert_eigengate_implements_consistent_protocols(
+            css.DDPowGate,
+            setup_code="import cirq_superstaq as css; import sympy",
+            ignoring_global_phase=True,
+        )
```

Comment:
might be wrong, but i think this mock isn't necessary if the gate doesn't define `_qasm_`
```suggestion
    cirq.testing.assert_eigengate_implements_consistent_protocols(
        css.DDPowGate,
        setup_code="import cirq_superstaq as css; import sympy",
        ignoring_global_phase=True,
    )
```